components/encumbered/rtmpdump/Makefile

```diff
@@ -14,30 +14,37 @@
 #
 
 BUILD_STYLE=justmake
-BUILD_BITS=64
-# Requires patching
-USE_OPENSSL11=no
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=         rtmpdump
-COMPONENT_VERSION=      2.4
-COMPONENT_REVISION=     2
+COMPONENT_VERSION=      2.6
 COMPONENT_SUMMARY=      RTMPdump -- a toolkit for RTMP streams
 COMPONENT_SRC=          $(COMPONENT_NAME)
-COMPONENT_ARCHIVE=      $(COMPONENT_NAME)-$(COMPONENT_VERSION).tar.gz
-COMPONENT_ARCHIVE_HASH= \
-	sha256:0be975011d116bbdf1befad2a984ac83d09b1061c86550326a7ec496d2f188fc
-COMPONENT_ARCHIVE_URL=  \
-	http://rtmpdump.mplayerhq.hu/download/$(COMPONENT_ARCHIVE)
 COMPONENT_PROJECT_URL=  http://rtmpdump.mplayerhq.hu/
 COMPONENT_LICENSE=      GPLv2,LGPLv2
 COMPONENT_LICENSE_FILE= rtmpdump.license
 COMPONENT_CLASSIFICATION=Applications/Sound and Video
 COMPONENT_FMRI=         video/rtmpdump
 
+GIT=git
```

Comment:
You do not need this.  `GIT` is already set to `/usr/bin/git` in `shared-macros.mk`.